metaverse_modules/sonic-boom/index.js

```diff
@@ -32,10 +32,10 @@ export default () => {
             //console.log(camera.rotation.y-localPlayer.rotation.y);
             //console.log(localPlayer.actionInterpolants.jump)
             currentPos.x=localPlayer.position.x;
-            currentPos.y=localPlayer.position.y;
+            //currentPos.y=localPlayer.position.y;
```

Comment:
Why is this no longer tracking Y position?

No mention of this in the description.

Reply:
This fixes trail stop.